addons/OPT/GPS/macros.hpp

```diff
@@ -1,2 +1,7 @@
 #define MODULE GPS
 #include "\opt\opt\addons\opt\macros.hpp"
+
+
+#define COLOR_SIDE			[0,0.3,0.6,1]
+#define COLOR_OWN_GROUP		[0.13, 0.54, 0.21, 1]
+#define COLOR_PLAYER_UNIT	[1, 0.4, 0, 1];
```

Comment:
Das Semikolon sollte da weg - könnte sonst fiese Fehler geben

Reply:
Quelle/Beispiele? Höre das immer wieder, aber meines Wissens nach galt das nur für SQS und gaaaaanz altes SQF. Seit spätestens A2 ist es egal, ob Fnc-Code mit Semikolon oder ohne endet.
https://community.bistudio.com/wiki/Function#Return_Values

Reply:
Beispiel:
```
if (_myColor == COLOR_PLAYER_UNIT) then {};
```

Reply:
Ach, das ";" wird im define mitgespeichert? Das erklärt es dann, ja. Danke! :)

Reply:
> Ach, das ";" wird im define mitgespeichert?

Jepp. In defines wird prinzipiell alles mitgespeichert, das kein Zeilenumbruch ist (von Leerzeichen zu nach dem Namen des Makros abgesehen) :)